### PR TITLE
Approve command versatile

### LINKED ISSUE #57: More flexible issue approval
## Description
As a user
I need to ensure more flexible issue approval
so that I can automate robust approval.

### Update [Required]
- [ ] Add stemming and lemmatization.
### Enhancement to Update [Optional]
- [ ] Potentially other improvements (synonyms, etc)

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by sammy1997**
# Description
Added more versatile approval command.

Fixes #57

# Type of Change: 
- Code
- New feature (non-breaking change which adds functionality pre-approved by mentors)

# How Has This Been Tested?
versatile_approval

# Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged 
- [ ] My changes generate no new warnings 
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: code/main_server.py [hunk 1]
```diff
@@ -284,5 +293,14 @@ def lemmatize_sent(sentence):
     return lemmatized_sentence
 
 
+def is_variant_of_approve(sentence):
+    stemmed_sentence_tokens = get_stems(sentence).split()
+    # All variants of approve like approve, approved, approving and approval have the same stem approv
+    if 'approv' in stemmed_sentence_tokens and 'no' not in stemmed_sentence_tokens \
```
Comment (sara-02) [inline code/main_server.py]:
```python
return (approv' in stemmed_sentence_tokens \
and 'no' not in stemmed_sentence_tokens \
and 'not' not in stemmed_sentence_tokens)
```

Comment (sara-02) [reply code/main_server.py]: Instead of using an if and 2 returns.

**Comment by sara-02** [comment]
@sammy1997 https://github.com/systers/sysbot/blob/8f49bc56465f591f31ba2cc70fddd381698ebb13/code/main_server.py#L300 Travis is complaining, I guess you need to a \ on the previous line.

**Comment by sammy1997** [comment]
@sara-02 That shouldn't be required if we have a bracket :/ . I will make an update, and see if it get's sorted

**Comment by sara-02** [comment]
@sammy1997 If it does not, then we can revert the old format for now, and try to fix the error later.
Feel free to revert to old logic in case the new one still gives linter issues.

**Pull request merged**